packages/itwin/one-click-lca-widget/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@itwin/one-click-lca-widget",
+  "version": "0.1.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Carbon Calculation.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Carbon Calculation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/itwin/viewer-components-react/tree/master/packages/itwin/one-click-lca-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/one-click-lca-widget.js",
+  "module": "lib/esm/one-click-lca-widget.js",
+  "typings": "lib/cjs/one-click-lca-widget",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": ""
```

Comment:
plz add some tests

Reply:
sorry, i know i said add tests, but i think certa here is probably overkill, couldve just gone with some unit tests i think, especially for the little test that you have.